sdks/python/apache_beam/coders/coders_test.py

```diff
@@ -109,6 +111,45 @@ def test_deterministic_proto_coder_determinism(self):
       self.assertEqual(coder.encode(mm_forward), coder.encode(mm_reverse))
 
 
+class ProtoPlusMessageB(proto.Message):
+  field1 = proto.Field(proto.BOOL, number=1)
+
+
+class ProtoPlusMessageA(proto.Message):
+  field1 = proto.Field(proto.STRING, number=1)
+  field2 = proto.RepeatedField(ProtoPlusMessageB, number=2)
+
+
+class ProtoPlusMessageWithMap(proto.Message):
+  field1 = proto.MapField(proto.STRING, ProtoPlusMessageA, number=1)
+
+
+class ProtoPlusCoderTest(unittest.TestCase):
+  def test_proto_plus_coder(self):
```

Comment:
Should we also add tests that include more complex field types (for example, maps) ?

Reply:
See line 140, this is tested.